cmd/vcert/validators.go

```diff
@@ -691,3 +693,86 @@ func validateSetPolicyFlags(commandName string) error {
 
 	return nil
 }
+
+func validateSshEnrollFlags(commandName string) error {
+
+	err := validateConnectionFlags(commandName)
+	if err != nil {
+		return err
+	}
+
+	if flags.sshCertKeyId == "" {
+		return fmt.Errorf("a ssh certificate key id is requiered")
```

Comment:
"An ID is required for the SSH certificate (--id)"

Reply:
fixed on most recent commit.